aws-auth-cognito/src/main/java/com/amplifyframework/statemachine/codegen/events/AuthenticationEvent.kt

```diff
@@ -38,7 +38,7 @@ class AuthenticationEvent(val eventType: EventType, override val time: Date? = n
         data class SignInRequested(val signInData: SignInData) : EventType()
         data class SignInCompleted(val signedInData: SignedInData) : EventType()
         data class SignOutRequested(val signOutData: SignOutData) : EventType()
-        data class CancelSignIn(val id: String = "") : EventType()
+        data class CancelSignIn(val id: String = "", val error: Exception? = null) : EventType()
```

Comment:
`id: String` can be removed since we added error. `id` exists because data class requires at least one parameter.